02_bubble/brnbke/main.go

```diff
@@ -0,0 +1,91 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
```

Comment:
Is the purpose of this purely to simplify testing?

Reply:
Yes pretty much, so I can change the output on subsequent Fprintln to "out" and grab that in the test.